rs/sns/governance/api/src/ic_sns_governance.pb.v1.rs

```diff
@@ -155,11 +155,20 @@ pub struct Neuron {
 }
 /// Nested message and enum types in `Neuron`.
 pub mod neuron {
```

Comment:
NCR

I think it is unusual for us to do "inline" mods; that is, they usually live in their own file.

Reply:
I think it was originally introduced by prost, since we started this file by copying the prost-generaed code. 

Closing this as it wasn't introduced in this PR.